snapcraft.yaml

```diff
@@ -0,0 +1,22 @@
+name: guetzli
+version: master
+summary: JPEG encoder for excellent compression density at high visual quality
+description: |
+  Guetzli-generated images are typically 20-30% smaller than images of
+  equivalent quality generated by libjpeg. Guetzli generates only sequential
+  (nonprogressive) JPEGs due to faster decompression speeds they offer.
+
+grade: devel
+confinement: strict
+
+apps:
+  guetzli:
+    command: bin/Release/guetzli
+    plugs: [home]
```

Comment:
Just curious: is there no way to say "this application has to access user-chosen files" and:
a) for commandline apps, consider paths mentioned on the commandline to be user-chosen
b) for GUI apps, require the app to use an external filepicker and consider files chosen in the filepicker as user-chosen?

It seems to kinda defeat the whole point of sandboxing if an app can modify ~/.bashrc